src/modules/apps/install.ts

```diff
@@ -108,11 +104,10 @@ export const prepareInstall = async (appsList: string[], force: boolean): Promis
   }
 }
 
-export default async (optionalApp: string, options) => {
+export default async (optionalApps: string[], options) => {
   const force = options.f || options.force
-  await validateAppAction('install', optionalApp)
-  const app = optionalApp || (await ManifestEditor.getManifestEditor()).appLocator
-  const appsList = [app, ...parseArgs(options._)]
+  await validateAppAction('install', optionalApps)
```

Comment:
`validateAppAction` in the way it works now expects a string as second argument, we'll have to change the second argument to something like `optionalApps.join(' ')`, or change `validateAppAction`